app/src/main/java/org/sagebionetworks/research/mpower/tracking/model/TrackingItem.java

```diff
@@ -0,0 +1,47 @@
+package org.sagebionetworks.research.mpower.tracking.model;
+
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+
+
+@AutoValue
```

Comment:
Can you give me a run down of what AutoValue does and why we used it for this project?  I'm new to it

Reply:
[Google's generator](https://github.com/google/auto/tree/master/value) for immutable POJOs with toString, hashCode, and equals. Auto-generates GSON adapters and Parcelable when we need it.